src/megameklab/com/ui/supportVeh/SVEquipmentDatabaseView.java

```diff
@@ -0,0 +1,125 @@
+import megameklab.com.ui.util.AbstractEquipmentDatabaseView;
+import megameklab.com.util.UnitUtil;
+import org.apache.logging.log4j.LogManager;
+
+import javax.swing.*;
+import java.util.Collection;
+import java.util.List;
+
+import static megameklab.com.ui.util.EquipmentTableModel.*;
+
+/**
+ * An Equipment Database for Support Vehicles. This table shows many columns
+ * and is suitable for use in the Equipment Tab.
+ */
+class SVEquipmentDatabaseView extends AbstractEquipmentDatabaseView {
+
+    private final List<Integer> fluffColumns = List.of(COL_NAME, COL_TECH, COL_TLEVEL, COL_TRATING, COL_DPROTOTYPE,
+            COL_DPRODUCTION, COL_DCOMMON, COL_DEXTINCT, COL_DREINTRO, COL_COST);
+
+    private final List<Integer> statsColumns = List.of(COL_NAME, COL_DAMAGE, COL_HEAT, COL_MRANGE, COL_RANGE,
+            COL_SHOTS, COL_TECH, COL_BV, COL_TON, COL_CRIT, COL_REF);
+
+    public SVEquipmentDatabaseView(EntitySource eSource) {
+        super(eSource);
+    }
+
+    @Override
+    protected void addEquipment(EquipmentType equip, int count) {
+        //TODO: This contains code for Large Craft. Not sure if needed for SV
+        Mounted mount;
+        boolean isMisc = equip instanceof MiscType;
+        try {
+            if (isMisc && equip.hasFlag(MiscType.F_TARGCOMP)) {
+                if (!UnitUtil.hasTargComp(eSource.getEntity())) {
+                    UnitUtil.updateTC(eSource.getEntity(), equip);
+                }
+            } else if (isMisc && UnitUtil.isFixedLocationSpreadEquipment(equip)) {
+                    int location = TestEntity.getSystemWideLocation(eSource.getEntity());
+                    mount = new Mounted(eSource.getEntity(), equip);
+                    eSource.getEntity().addEquipment(mount, location, false);
+            } else {
+                if (equip instanceof AmmoType) {
+                    if (eSource.getEntity().usesWeaponBays()) {
+                        addLargeCraftAmmo(equip, count);
+                        return;
+                    } else if (eSource.getEntity().isAero()) {
+                        addBodyAmmo(equip, count, Aero.LOC_FUSELAGE);
+                        return;
+                    } else if (eSource.getEntity() instanceof Tank) {
+                        addBodyAmmo(equip, count, Tank.LOC_BODY);
+                        return;
+                    }
+                }
+                for (int i = 0; i < count; i++) {
+                    mount = new Mounted(eSource.getEntity(), equip);
+                    if ((eSource.getEntity().isFighter()
+                            && equip instanceof MiscType) && equip.hasFlag(MiscType.F_BLUE_SHIELD)) {
+                        getAero().addEquipment(mount, Aero.LOC_FUSELAGE, false);
+                    } else {
+                        eSource.getEntity().addEquipment(mount, Entity.LOC_NONE, false);
+                    }
+                    if ((equip instanceof WeaponType) && (equip.hasFlag(WeaponType.F_ONESHOT)
```

Comment:
Can we have spaces between if statements if they aren't supposed to be chained? It's a misread prevention strategy.